### ISSUE TITLE
Backup doesn't put all my entries and I cannot restore on other forks

### ISSUE THREAD

**Issue by edmxn** [null2264/yokai#49]
### Steps to reproduce

Create backup file
Restore in other fork with it

### Expected behavior

Should import/restore everything with no problem

### Actual behavior

More than 300 entries are lost

### Crash logs

The app never gives me

### Yōkai version

1.7.74

### Android version

Android 12

### Device

Samsung S10+

### Other details

I've been using the app for quite some time now, but due to the amount of lag and crashes I decided to stop using it and migrate
But, when trying to restore, in the end I always lose things, this because yokai doesn't backup em all well, leaving things behind

I have in total 5989 entries in my library, and only like 5600 reach to restore in other forks
In TSY it tells me that it failed with 80 and only managed to restore 5600, when it doesn't even come close with those 80 to what I actually have (5989)

I know I have quite a few entries (maybe, I don't really know) but that's no excuse

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, or a request for an extension, I should be contacting the extensions repository's maintainer/support for help.
- [X] I am reporting an issue exclusive to this fork. I have also checked that is not an issue on the [main version of Mihon](https://github.com/mihonapp/mihon)
- [X] I have tried the [troubleshooting guide](https://mihon.app/help/).
- [X] I have updated the app to version **[1.7.14](https://github.com/null2264/yokai/releases/latest)**.
- [X] I have updated all installed extensions.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I have filled out all of the requested information in this form.

**Comment by lalalasupa0** [comment]
After some time it did gave me a good backup file.

**Comment by null2264** [comment]
Yeah, I was debugging this issue and can't seem to reproduce this issue, my initial thought was that scanlators filter is not compatible with Mihon's, but it works fine when I test it. Not sure what went wrong before...
